### DISCUSSION TITLE
Reviews from the Software Manager

### DISCUSSION THREAD

**Discussion by mortim** [linuxmint/discussions#545] [Questions] [answered]
Hi,
I was wondering why Linux Mint removed Unverified Flatpaks reviews. It seems to be a bad decision to me.

What's your opinion ?

**Comment by phuocthanhlamnguyen-gif** [comment, marked as answer]
It's mainly to prevent user trust from being falsely built on unverified software, but without reviews, the software may contain viruses, which is not a good time for the user, so it depends on what you see. If you ask about my opinion, then well, it's also pretty bad for me

**Reply by mortim** [reply]
Actually, the reason that you mentioned above is quite fair. As long as the Mint Software Manager does not show the unverified flatpaks by default it's not a big deal in terms of security for new users. The user should be fully aware if he wants to enable those flatpaks.

Thx for the reply.

**Reply by phuocthanhlamnguyen-gif** [reply]
Also as long as they don't turn on the setting to show the unverified flatpaks

**Reply by mortim** [reply]
> Also as long as they don't turn on the setting to show the unverified flatpaks

Yeah that was I tried to say in my last sentence, it wasn't very clear sry.

**Reply by phuocthanhlamnguyen-gif** [reply]
It's fine
